tests/FakeItEasy.Tests/Core/CallOrderingContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core

```

Comment:
Probably?
Sounds good. Me, I'd hope that they never had to use anything but `FakeItEasy`, not that I'd be against putting super advanced but publicly accessible features in another namespace sometime…